less/release-9.less

```diff
@@ -0,0 +1,309 @@
+      }
+      text-transform: uppercase;
+      line-height: 1;
+      margin-bottom: 0;
+      &:after {
+        border-bottom:none;
+      }
+    }
+  
+    p, p span {
+      font-weight: 400;
+      text-transform: uppercase;
+      font-stretch: condensed;
+    }
+  
+    .jumbotron.featured-jumbotron-download {
+      min-height: 650px;
+      background-image:url("images/release/jakarta9-header.jpg");
+      background-repeat:no-repeat;
+      background-size:cover;
+  
+      .btn-primary {
+        color: @white;
+        background-color: transparent;
+        border-color: @white;
+        min-height: 70px;
+        text-transform: uppercase;
+      }
+
+      .btn-primary:hover {
+        color: @brand-primary;
+      }
+    }
+  
+    .jumbotron-right-content {
+      font-family: 'Exo', sans-serif;
+      font-size: 150px;
+      @media (min-width: @screen-sm-min) {
+        font-size: 300px;
+      }
+      @media (min-width: @screen-md-min) {
+        font-size: 300px;
+      }
+      @media (min-width: @screen-lg-min) {
+        font-size: 300px;
+      }
+      line-height: 1;
+      font-weight: 400;
+    }
+    ul.buttons {
+      li {
+        width:100%;
+        @media (min-width: @screen-md-min) {
+          width:49%;
+        }
+      }
+    }
+    .btn {
+      width:100%;
+      @media (max-width: @screen-sm-max) {
+        margin-bottom:15px;
+      }
+    }
+}
+  
+
+.features-and-benefits {
+    padding:30px 0 60px 0;
+
+    .feature-box {
+        background-color: #eeeded;
+        margin: 10px;
+        padding: 0 40px;
+        border-top-left-radius: 30px;
+        border-top-right-radius: 30px;
+        border-bottom-left-radius: 5px;
+        border-bottom-right-radius: 5px;
+        -webkit-box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+        -moz-box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+        box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+
+        h3 {
+            padding-top: 70px;
+        }
+
+        p {
+            padding-bottom: 30px;
+        }
+
+        hr {
+            max-width: 150px;
+            border-top: 5px solid @brand-primary;
+            margin-left: 0;
+        }
+
+    }
+
+    .polygons {
+        background: @brand-secondary;
+        height: 75px;
+        position: relative;
+        width: 70px;
+
+    }
+
+    .polygons:after {
+        border-top: 10px solid @brand-secondary;
+        border-left: 35px solid transparent;
+        border-right: 35px solid transparent;
+        content: "";
+        height: 0;
+        left: 0;
+        position: absolute;
+        top: 75px;
+        width: 0;
+    }
+
+    .polygons-right {
+        float: right;
+        margin-top: -10px;
+    }
+
+}
+
+.jakartaee-hr {
+    max-width: 150px;
+    border-top: 5px solid @brand-primary;
+    margin-left: 0;
+}
+
+
+.font-numbers {
+
+    @media (min-width: @screen-sm-min) {
+      font-size: 100px;
+    }
+  
+    @media (max-width: @screen-xs-max) {
```

Comment:
For example, 
for this one, I just want the number to be bigger on desktop, and smaller on mobile. 
So that you mean I do not need to have "@media (max-width: @screen-xs-max) ", just use "@media (min-width: @screen-sm-min) " ? 

```
.font-numbers {

    @media (min-width: @screen-sm-min) {
      font-size: 100px;
    }
  
    @media (max-width: @screen-xs-max) {
      font-size: 65px;
    }
}

```

Reply:
Wait, isn't this mobile-first?  @media (max-width: @screen-xs-max)

Reply:
Got it, make mobile-first instead.